docs/source/documentation/debug.rst

```diff
@@ -50,6 +50,14 @@ Two local modes are available:
 
 The subprocess mode is much faster than the Docker mode, but does not test that the Dockerfiles of the assets are valid, and may fail if advanced COPY or ADD commands are used in the Dockerfile. Therefore, the Docker mode is closer to the remote mode than the subprocess mode. It is recommended to run your experiment locally in subprocess mode and when it is ready, then test it with the Docker mode.
 
+.. note::
+    For the **Docker mode on Windows**, follow these `instructions <https://docs.docker.com/desktop/install/windows-install/>`__ to install Docker desktop.
+
+    Once installed:
+
+        * In ``Settings`` -> ``General``, check the ``Expose daemon on tcp://localhost:2375 without TLS`` option.
+        * Set the environment variable ``DOCKER_HOST`` to  ``tcp://localhost:2375``
```

Comment:
Why do we need to do that?

Reply:
And should write the reason here?

Reply:
From what I found, by default, you don't have access to the dcker daemon on windows. Either you do this, either you configure your docker to keep the TLS activated.

Reply:
Ok that's fine.
Let's dig into that later, at least we have a way of making it work!